### PR TITLE
[AMP Stories] Add "Tip" block style for Text Block.

### PR DESCRIPTION
**Pull request by miina**
See #2013.

### CODE REVIEW

File: assets/css/amp-stories.css [hunk 1]
```diff
@@ -4,9 +4,11 @@ amp-story-grid-layer[template="fill"] .wp-block-image { margin: 0; }
...
-.is-style-rounded {
+.is-style-rounded,
+.is-style-rounded img {
     border-radius: 5px;
 }
-.is-style-rounded img {
-	border-radius: 5px;
+
+.is-style-tip {
+	border-radius: 3pt 3pt 20pt 20pt;
```
Comment (swissspidy) [inline assets/css/amp-stories.css]: I thought `pt` is more suited for print than screen. What's the use case here? Just wanna understand it better.

Comment (miina) [reply assets/css/amp-stories.css]: These values are taken from design. We can convert to pixels, too.

Comment (swissspidy) [reply assets/css/amp-stories.css]: Then let's do it :-)

**Comment by miina** [comment]
Changed to "Half Rounded" :D 
It is more descriptive, the word "Tip" without context is not clear.

**Comment by westonruter** [comment]
Should the CSS have a `::first-line` rule that makes the first line a different style, like smaller font size?

**Comment by miina** [comment]
Maybe it would be okay to use another Text Block in case of wanting to have different font size? This way the "Half Rounded" could be used more universally.

**Pull request merged**